.maintain/monitoring/grafana-dashboards/README_dashboard.md

```diff
@@ -5,3 +5,10 @@ Shared templated Grafana dashboards.
 To import the dashboards follow the [Grafana
 documentation](https://grafana.com/docs/grafana/latest/reference/export_import/).
 You can see an example setup [here](../../../.maintain/sentry-node).
+
+#### Required labels on Prometheus metrics
```

Comment:
This was deleted in master :D 

And the file below as well :D

Reply:
oh shit, it's all a mess now. :(

Reply:
That is why I prefer to resolve conflicting commits from master one by one :grin: https://github.com/cecton/git-tools#git-try-merge